sql-ballerina/Module.md

```diff
@@ -0,0 +1,302 @@
+## Overview
+
+This module provides the common interface and functionality to interact with a database. The corresponding database
+clients can be created by using specific database modules such as `MySQL` or using the Java Database Connectivity 
+module `JDBC`.
+
+### List of Database Modules
+Ballerina now has [JDBC module](https://ballerina.io/learn/api-docs/ballerina/#/jdbc) as the generic db connector module to connect with any relational database by simply providing the JDBC URL and the other related properties.
```

Comment:
```suggestion
Ballerina now has the [`jdbc` module](https://ballerina.io/learn/api-docs/ballerina/#/jdbc) as the generic DB connector module to connect to any relational database by simply providing the JDBC URL and the other related properties.
```

Reply:
Added suggested change in https://github.com/ballerina-platform/module-ballerina-sql/pull/228/commits/fa55a74e7928c9529090430cd4a5a358e2af8e8e